cmd/lotus-fountain/site/miner.html

```diff
@@ -27,6 +27,9 @@
         <div class="Index-node">
             <span>When creating storage miner, DO NOT REFRESH THE PAGE, wait for it to load. This can take more than 5min.</span>
         </div>
+        <div class="Index-node">
+            <span>If you don't have an owner/worker address, you can create it by following <a target="_blank" href="https://docs.lotu.sh/en+mining#get-started-22083">these instructions</a>.</span>
```

Comment:
I don't think this link will actually stick to the "getting started" section? Maybe just say "you can create it by running `lotus wallet new bls`"?